### PR TITLE
added Python cheat sheets

### PR DESCRIPTION
**Pull request by Ramona-Saintandre**
Thanks for hosting this.
Just getting back into coding, and since I have been looking for resources for Python. 
I thought I would add what I have found. 

Happy hacking

## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [ ] Not a duplicate
- [ ] Included author(s) if appropriate
- [ ] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-cheatsheets.md [hunk 1]
```diff
@@ -109,7 +109,8 @@
 * [Python for Data Science Cheatsheet](https://pydatascience.org/data-science-cheatsheets/) - DataCamp (HTML and Image)
 * [Python Regex Cheatsheet](https://www.debuggex.com/cheatsheet/regex/python) - Debuggex (HTML)
 * [Python WorldWide Cheatsheet](https://cheatography.com/davechild/cheat-sheets/python/) - Cheatography (Python)
-
+* [Python Cheat Sheet] (https://websitesetup.org/python-cheat-sheet/)
```
Comment (SethiPandi) [inline free-programming-cheatsheets.md:112]: Could you please remove the space between the `]` and `(`?

```suggestion
* [Python Cheat Sheet](https://websitesetup.org/python-cheat-sheet/)
```

File: free-programming-cheatsheets.md [hunk 2]
```diff
@@ -109,7 +109,8 @@
...
+* [Comprehensive Python Cheat Sheet for Beginners] (https://medium.com/the-codehub/comprehensive-python-cheat-sheet-for-beginners-5d76bb038fa2)
```
Comment (SethiPandi) [inline free-programming-cheatsheets.md:113]:
```suggestion
* [Comprehensive Python Cheat Sheet for Beginners](https://medium.com/the-codehub/comprehensive-python-cheat-sheet-for-beginners-5d76bb038fa2)
```

**Comment by eshellman** [comment]
alphabetization, newlines.
Be sure to check the output of Travis-CI for linter errors. Alphabetical order and spacing are the most common problems.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**